app/services/return-requirements/submit-additional-submission-options.service.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+/**
+ * Orchestrates validating the data for `/return-requirements/{sessionId}/additional-submission-options` page
+ * @module AdditionalSubmissionOptionsService
+ */
+
+const AdditionalSubmissionOptionsPresenter = require('../../presenters/return-requirements/additional-submission-options.presenter.js')
+const AdditionalSubmissionOptionsValidator = require('../../validators/return-requirements/additional-submission-options.validator.js')
+const SessionModel = require('../../models/session.model.js')
+
+/**
+ * Orchestrates validating the data for `/return-requirements/{sessionId}/additional-submission-options` page
+ *
+ * It first retrieves the session instance for the returns requirements journey in progress.
+ *
+ * The validation result is then combined with the output of the presenter to generate the page data needed by the view.
+ * If there was a validation error the controller will re-render the page so needs this information. If all is well the
+ * controller will redirect to the next page in the journey.
+ *
+ * @param {string} sessionId - The id of the current session
+ * @param {Object} payload - The submitted form data
+ * @param {Object} yar - The Hapi `request.yar` session manager passed on by the controller
+ *
+ * @returns {Promise<Object>} If no errors it returns an empty object else the page data for the note page including the
+ * validation error details
+ */
+async function go (sessionId, payload, yar) {
+  const session = await SessionModel.query().findById(sessionId)
+
+  _handleOneOptionSelected(payload)
+
+  const validationResult = _validate(payload)
+
+  if (!validationResult) {
+    const notification = _notification(session, payload)
+    await _save(session, payload)
+
+    if (notification) {
+      yar.flash('notification', notification)
+    }
+
+    return {}
+  }
+
+  const submittedSessionData = _submittedSessionData(session, payload)
+
+  return {
+    activeNavBar: 'search',
+    error: validationResult,
+    pageTitle: 'Select any additional submission options for the return requirements',
+    ...submittedSessionData
+  }
+}
+
+/**
+ * When a single agreement and exception is checked by the user, it returns as a string. When multiple agreements and
+ * exceptions are checked, the 'additionalSubmissionOptions' is returned as an array.
+ * This function works to make those single
+ * selected string 'additionalSubmissionOptions' into an array for uniformity.
```

Comment:
Just needs an update after the copy and paste.

```suggestion
 * When a single additional submission option is checked by the user, it returns as a string. When multiple options are
 * checked, the 'additionalSubmissionOptions' is returned as an array. This function works to make those single selected
 * string 'additionalSubmissionOptions' into an array for uniformity.
```